manifests/h/Huawei/WeMeeting/1.2.0.5/Huawei.WeMeeting.locale.zh-CN.yaml

```
@@ -0,0 +1,17 @@
+# Created using wingetcreate 0.5.0.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: Huawei.WeMeeting
+PackageVersion: 1.2.0.5
+PackageLocale: zh-CN
+Publisher: Huawei Technologies Co., Ltd.

```

```suggestion
Publisher: Huawei Technologies CO., LTD
```
This must match Control Panel.